apps/site/src/lib/components/forms/shared/ApplicationFlow.tsx

```diff
@@ -16,14 +16,14 @@ interface ApplicationFlowProps {
 		prefaceAccepted?: string;
 	};
 	applicationType: "Hacker" | "Mentor" | "Volunteer";
-	continueHREF: string;
+	applicationURL: string;
```

Comment:
Suggestion: might be good to enforce the possible values as a union
```suggestion
	applicationURL: "/apply" | "/mentor" | "/volunteer";
```